No module named markupsafe in one liners

Development and setup one liners has the following error

```
ERROR! Unexpected Exception: No module named markupsafe
the full traceback was:

Traceback (most recent call last):
  File "/tmp/ansible/bin/ansible-playbook", line 73, in <module>
    mycli = getattr(__import__("ansible.cli.%s" % sub, fromlist=[myclass]), myclass)
  File "/tmp/ansible/lib/ansible/cli/playbook.py", line 30, in <module>
    from ansible.executor.playbook_executor import PlaybookExecutor
  File "/tmp/ansible/lib/ansible/executor/playbook_executor.py", line 27, in <module>
    from ansible.executor.task_queue_manager import TaskQueueManager
  File "/tmp/ansible/lib/ansible/executor/task_queue_manager.py", line 28, in <module>
    from ansible.executor.play_iterator import PlayIterator
  File "/tmp/ansible/lib/ansible/executor/play_iterator.py", line 29, in <module>
    from ansible.playbook.block import Block
  File "/tmp/ansible/lib/ansible/playbook/__init__.py", line 25, in <module>
    from ansible.playbook.play import Play
  File "/tmp/ansible/lib/ansible/playbook/play.py", line 27, in <module>
    from ansible.playbook.base import Base
  File "/tmp/ansible/lib/ansible/playbook/base.py", line 32, in <module>
    from jinja2.exceptions import UndefinedError
  File "/usr/local/lib/python2.7/dist-packages/jinja2/__init__.py", line 33, in <module>
    from jinja2.environment import Environment, Template
  File "/usr/local/lib/python2.7/dist-packages/jinja2/environment.py", line 13, in <module>
    from jinja2 import nodes
  File "/usr/local/lib/python2.7/dist-packages/jinja2/nodes.py", line 19, in <module>
    from jinja2.utils import Markup
  File "/usr/local/lib/python2.7/dist-packages/jinja2/utils.py", line 531, in <module>
    from markupsafe import Markup, escape, soft_unicode
```
This is possible because of jinja2 dependency update and this module is not present in the system.

Also higher in the log there is following error

```
Running setup.py install for cryptography
    c/_cffi_backend.c:15:17: fatal error: ffi.h: No such file or directory
     #include <ffi.h>
                     ^
    compilation terminated.
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip_build_root/cryptography/setup.py", line 335, in <module>
        **keywords_with_side_effects(sys.argv)
      File "/usr/lib/python2.7/distutils/core.py", line 111, in setup
        _setup_distribution = dist = klass(attrs)
      File "/usr/lib/python2.7/dist-packages/setuptools/dist.py", line 239, in __init__
        self.fetch_build_eggs(attrs.pop('setup_requires'))
      File "/usr/lib/python2.7/dist-packages/setuptools/dist.py", line 264, in fetch_build_eggs
        replace_conflicting=True
      File "/usr/lib/python2.7/dist-packages/pkg_resources.py", line 620, in resolve
        dist = best[req.key] = env.best_match(req, ws, installer)
      File "/usr/lib/python2.7/dist-packages/pkg_resources.py", line 858, in best_match
        return self.obtain(req, installer) # try and download/install
      File "/usr/lib/python2.7/dist-packages/pkg_resources.py", line 870, in obtain
        return installer(requirement)
      File "/usr/lib/python2.7/dist-packages/setuptools/dist.py", line 314, in fetch_build_egg
        return cmd.easy_install(req)
      File "/usr/lib/python2.7/dist-packages/setuptools/command/easy_install.py", line 616, in easy_install
        return self.install_item(spec, dist.location, tmpdir, deps)
      File "/usr/lib/python2.7/dist-packages/setuptools/command/easy_install.py", line 646, in install_item
        dists = self.install_eggs(spec, download, tmpdir)
      File "/usr/lib/python2.7/dist-packages/setuptools/command/easy_install.py", line 834, in install_eggs
        return self.build_and_install(setup_script, setup_base)
      File "/usr/lib/python2.7/dist-packages/setuptools/command/easy_install.py", line 1040, in build_and_install
        self.run_setup(setup_script, setup_base, args)
      File "/usr/lib/python2.7/dist-packages/setuptools/command/easy_install.py", line 1028, in run_setup
        raise DistutilsError("Setup script exited with %s" % (v.args[0],))
    distutils.errors.DistutilsError: Setup script exited with error: command 'x86_64-linux-gnu-gcc' failed with exit status 1
    Complete output from command /usr/bin/python -c "import setuptools, tokenize;__file__='/tmp/pip_build_root/cryptography/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /tmp/pip-C_YpDh-record/install-record.txt --single-version-externally-managed --compile:
    c/_cffi_backend.c:15:17: fatal error: ffi.h: No such file or directory

 #include <ffi.h>

                 ^
```
This might cause markupsafe error or they are two separate bugs need to be checked.

--- a/bin/setup_dev_machine
+++ b/bin/setup_dev_machine
@@ -112,6 +112,8 @@ rm -rf sr-build-tools &
 rm -rf ansible &
 wait
 
+# This is only necessary for non-desktop images
+sudo apt-get install -y python-crypto
 # These are necessary dependencyes for the cryptography python package that is a dependency of paramiko (see http://www.paramiko.org/installing.html)
 sudo apt-get install -y build-essential libssl-dev libffi-dev python-dev
 sudo apt-get remove ansible -y &

--- a/bin/setup_dev_machine
+++ b/bin/setup_dev_machine
@@ -107,17 +107,13 @@ echo ""
 
 sudo apt-get update
 
-sudo apt-get install -y python-pip git libyaml-dev libpython2.7-dev &
+# python-crypto is only necessary for non-desktop images
+# build-essential libssl-dev libffi-dev python-dev are necessary dependencies for the cryptography python package that is a dependency of paramiko (see http://www.paramiko.org/installing.html)
+sudo apt-get install -y python-pip git libyaml-dev libpython2.7-dev python-crypto build-essential libssl-dev libffi-dev python-dev &
 rm -rf sr-build-tools &
 rm -rf ansible &
 wait
 
-# This is only necessary for non-desktop images
-sudo apt-get install -y python-crypto &
-wait
-# These are necessary dependencies for the cryptography python package that is a dependency of paramiko (see http://www.paramiko.org/installing.html)
-sudo apt-get install -y build-essential libssl-dev libffi-dev python-dev &
-wait
 sudo apt-get remove ansible -y &
 sudo pip install paramiko PyYAML Jinja2 httplib2 six &
 git clone --branch 'v2.0.1.0-0.2.rc2' git://github.com/ansible/ansible.git --recursive &

--- a/bin/setup_dev_machine
+++ b/bin/setup_dev_machine
@@ -113,9 +113,11 @@ rm -rf ansible &
 wait
 
 # This is only necessary for non-desktop images
-sudo apt-get install -y python-crypto
-# These are necessary dependencyes for the cryptography python package that is a dependency of paramiko (see http://www.paramiko.org/installing.html)
-sudo apt-get install -y build-essential libssl-dev libffi-dev python-dev
+sudo apt-get install -y python-crypto &
+wait
+# These are necessary dependencies for the cryptography python package that is a dependency of paramiko (see http://www.paramiko.org/installing.html)
+sudo apt-get install -y build-essential libssl-dev libffi-dev python-dev &
+wait
 sudo apt-get remove ansible -y &
 sudo pip install paramiko PyYAML Jinja2 httplib2 six &
 git clone --branch 'v2.0.1.0-0.2.rc2' git://github.com/ansible/ansible.git --recursive &

--- a/bin/setup_dev_machine
+++ b/bin/setup_dev_machine
@@ -112,6 +112,8 @@ rm -rf sr-build-tools &
 rm -rf ansible &
 wait
 
+# These are necessary dependencyes for the cryptography python package that is a dependency of paramiko (see http://www.paramiko.org/installing.html)
+sudo apt-get install -y build-essential libssl-dev libffi-dev python-dev
 sudo apt-get remove ansible -y &
 sudo pip install paramiko PyYAML Jinja2 httplib2 six &
 git clone --branch 'v2.0.1.0-0.2.rc2' git://github.com/ansible/ansible.git --recursive &

--- a/bin/setup_production_machine
+++ b/bin/setup_production_machine
@@ -22,6 +22,8 @@ rm -rf sr-build-tools &
 rm -rf ansible &
 wait
 
+# This is only necessary for non-desktop images
+sudo apt-get install -y python-crypto
 # These are necessary dependencyes for the cryptography python package that is a dependency of paramiko (see http://www.paramiko.org/installing.html)
 sudo apt-get install -y build-essential libssl-dev libffi-dev python-dev
 sudo apt-get remove ansible -y &

--- a/bin/setup_production_machine
+++ b/bin/setup_production_machine
@@ -17,15 +17,13 @@ echo ""
 
 sudo apt-get update
 
-sudo apt-get install -y python-pip git libyaml-dev libpython2.7-dev &
+# python-crypto is only necessary for non-desktop images
+# build-essential libssl-dev libffi-dev python-dev are necessary dependencies for the cryptography python package that is a dependency of paramiko (see http://www.paramiko.org/installing.html)
+sudo apt-get install -y python-pip git libyaml-dev libpython2.7-dev python-crypto build-essential libssl-dev libffi-dev python-dev &
 rm -rf sr-build-tools &
 rm -rf ansible &
 wait
 
-# This is only necessary for non-desktop images
-sudo apt-get install -y python-crypto
-# These are necessary dependencyes for the cryptography python package that is a dependency of paramiko (see http://www.paramiko.org/installing.html)
-sudo apt-get install -y build-essential libssl-dev libffi-dev python-dev
 sudo apt-get remove ansible -y &
 sudo pip install paramiko PyYAML Jinja2 httplib2 six &
 git clone --branch 'v2.0.1.0-0.2.rc2' git://github.com/ansible/ansible.git --recursive &

--- a/bin/setup_production_machine
+++ b/bin/setup_production_machine
@@ -22,6 +22,8 @@ rm -rf sr-build-tools &
 rm -rf ansible &
 wait
 
+# These are necessary dependencyes for the cryptography python package that is a dependency of paramiko (see http://www.paramiko.org/installing.html)
+sudo apt-get install -y build-essential libssl-dev libffi-dev python-dev
 sudo apt-get remove ansible -y &
 sudo pip install paramiko PyYAML Jinja2 httplib2 six &
 git clone --branch 'v2.0.1.0-0.2.rc2' git://github.com/ansible/ansible.git --recursive &

